docs/README.md

```diff
@@ -316,6 +316,7 @@ In order to apply CSS customizations in this and other blocks, follow the instru
 | `paymentGroup`   |
 | `paymentValue`   |
 | `addressContainer`   |
+| `locationName`   |
```

Comment:
Hi, thanks for your contribution. Maybe locationName is too generic, since it only applies what do you think of using the name of that field instead?
```suggestion
| `locationName`   |
```
```suggestion
| `pickupFriendlyName`   |
```